librz/core/cil.c

```diff
@@ -367,7 +367,9 @@ RZ_IPI void rz_core_analysis_esil_emulate(RzCore *core, ut64 addr, ut64 until_ad
 
 RZ_IPI void rz_core_analysis_esil_emulate_bb(RzCore *core) {
 	RzAnalysisBlock *bb = rz_analysis_find_most_relevant_block_in(core->analysis, core->offset);
-	rz_core_analysis_esil_emulate(core, bb->addr, UT64_MAX, bb->ninstr);
+	if (bb) {
+		rz_core_analysis_esil_emulate(core, bb->addr, UT64_MAX, bb->ninstr);
+	}
```

Comment:
I would do the opposite and check `if (!bb) { ... print something + return error ... }` otherwise if i try to emulate a basicblock and see no output, as a user, i have no idea if things went well or not and why.